### PR TITLE
Add clue benchmark classification

### PR DESCRIPTION
**Pull request by LiuChiachi**
### PR types

New features
### PR changes

Models
### Description

run_clue.py for classification

mrc, chid, c3 are from https://github.com/PaddlePaddle/PaddleNLP/pull/1722.

### CODE REVIEW

File: examples/benchmark/clue/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+eval loss: 2.762143, acc: 0.0532, eval done total : 9.460448503494263 s
+global step 200/10008, epoch: 0, batch: 199, rank_id: 0, loss: 2.536534, lr: 0.0000060000, speed: 5.6834 step/s
+eval loss: 2.326450, acc: 0.26, eval done total : 9.412081480026245 s
+global step 300/10008, epoch: 0, batch: 299, rank_id: 0, loss: 1.847109, lr: 0.0000090000, speed: 5.2913 step/s
+eval loss: 1.447455, acc: 0.471, eval done total : 9.519582033157349 s
+```
+
+使用各种预训练模型进行 Fine-tuning，在 CLUE 的各验证集上有如下结果：
+
+| CLUE Score            | AFMQC   | TNEWS   | IFLYTEK | CMNLI   | OCNLI   | CLUEWSC2020 | CSL     | CMRC2018 | CHID    | C3      |
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: CLUE Score -> 模型？更合适一些

File: examples/benchmark/clue/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# CLUE Benchmark
+
+[CLUE](https://www.cluebenchmarks.com/)自成立以来发布了多项 NLP 评测基准，包括分类榜单，阅读理解榜单和自然语言推断榜单等，在学术界、工业界产生了深远影响。
+是目前应用最广泛的中文语言测评指标之一，被包括阿里巴巴达摩院、腾讯 AI 实验室、华为诺亚方舟实验室在内的 20 多家国内语言实验室所采纳。学术引用 100+，github star 超 6000+。详细可参考 [CLUE论文](https://arxiv.org/abs/2004.05986)
+
+本项目是 CLUE 评测任务 在 Paddle 2.2.2 上的开源实现。
+
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md:6]: 是否在这里先整体展示评测结果表格？表格下方描述清楚评测方式。然后再接快速开始教用户如何评测？

**Review by tianxin1860 — CHANGES_REQUESTED** [review]
README 逻辑建议稍微调整下：先展示评测结果，再教用户如何一件运行评测。

File: examples/benchmark/clue/README.md (hunk 2 above)
Comment (LiuChiachi) [reply examples/benchmark/clue/README.md:6]: 感谢建议，已经修改:)

File: examples/benchmark/clue/README.md (hunk 1 above)
Comment (LiuChiachi) [reply examples/benchmark/clue/README.md]: Thanks， 已经修改了

File: examples/benchmark/clue/README.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+# CLUE Benchmark
+
+[CLUE](https://www.cluebenchmarks.com/)自成立以来发布了多项 NLP 评测基准，包括分类榜单，阅读理解榜单和自然语言推断榜单等，在学术界、工业界产生了深远影响。
+是目前应用最广泛的中文语言测评指标之一，被包括阿里巴巴达摩院、腾讯 AI 实验室、华为诺亚方舟实验室在内的 20 多家国内语言实验室所采纳。学术引用 100+，github star 超 6000+。详细可参考 [CLUE论文](https://arxiv.org/abs/2004.05986)
+
+本项目是 CLUE 评测任务 在 PaddlePaddle 2.0 上的开源实现。
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: 本项目基于 PaddlePaddle 在 CLUE 数据集上对领先的开源预训练模型模型进行了充分评测，为开发者在预训练模型选择上提供参考，同时开发者基于本项目可以轻松一键复现模型效果，也可以参加 CLUE 竞赛取得好成绩。

File: examples/benchmark/clue/README.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+# CLUE Benchmark
+
+[CLUE](https://www.cluebenchmarks.com/)自成立以来发布了多项 NLP 评测基准，包括分类榜单，阅读理解榜单和自然语言推断榜单等，在学术界、工业界产生了深远影响。
+是目前应用最广泛的中文语言测评指标之一，被包括阿里巴巴达摩院、腾讯 AI 实验室、华为诺亚方舟实验室在内的 20 多家国内语言实验室所采纳。学术引用 100+，github star 超 6000+。详细可参考 [CLUE论文](https://arxiv.org/abs/2004.05986)
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: `****是目前最广泛的语言评测指标之一。` 这句话之后的描述删掉吧。

File: examples/benchmark/clue/README.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+本项目是 CLUE 评测任务 在 PaddlePaddle 2.0 上的开源实现。
+
+## 整体结果
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: CLUE 评测结果

File: examples/benchmark/clue/README.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+使用ERNIE 3.0 Base 和 RoBERTa-wwm-ext-large 进行 Fine-tuning，在 CLUE 的各验证集上有如下结果：
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: 这些信息在表格里已经有体现，删掉吧。

File: examples/benchmark/clue/README.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+| Model                 | AFMQC   | TNEWS   | IFLYTEK | CMNLI   | OCNLI   | CLUEWSC2020 | CSL     | CMRC2018 | CHID    | C3      |
+| --------------------- | ------- | ------- | ------- | ------- | ------- | ----------- | ------- | -------- | ------- | ------- |
+| Metric                | dev/Acc | dev/Acc | dev/Acc | dev/Acc | dev/Acc | dev/Acc     | dev/Acc | dev/F1   | dev/Acc | dev/Acc |
+| ERNIE 3.0 Base        | 75.86   | 58.92   | 61.18   | 83.35   | 79.66   | 86.51       | 82.73   | 89.48    |         |         |
+| RoBERTa-wwm-ext-large | 76.20   | 59.50   | 62.10   | 84.02   | 79.15   | 90.79       | 82.03   |          |         |         |
+
+**NOTE：**
+
+1.上表所有模型的精度测试均是基于下方超参数范围进行的 Grid Search 超参寻优。在每个配置下训练时，每隔 100 个 steps 在验证集上评估一次，取验证集上最佳准确率作为当前超参数配置下的准确率；
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: **NOTE:具体评测方式如下**
1. 以上所有任务均基于 Grid Search 方式进行超参寻优，训练每间隔 100 steps 评估验证集效果，取验证集最优效果作为表格中的汇报指标。
2. Grid Search 超参范围:  batch_size: 16, 32, 64; learning rates: 1e-5, 2e-5, 3e-5, 5e-5
3. 因为 CLUEWSC2020 数据集效果对 batch_size 较为敏感，CLUEWSC2020 评测时额外增加了 batch_size = 8 的超参搜索。

File: examples/benchmark/clue/README.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+- batch sizes: 16, 32, 64;
+- learning rates: 1e-5, 2e-5, 3e-5, 5e-5
+
+其中单独为 CLUEWSC2020 任务添加了 batch_size = 8 的实验，
+
+## 快速开始
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md]: ## 一键复现模型效果

Comment (tianxin1860) [reply examples/benchmark/clue/README.md]: 这一小节以 TNEWS 任务为例教用户怎样一键复现我们的评测结果

File: examples/benchmark/clue/README.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+Fine-tuning过程将按照 `logging_steps` 和 `save_steps` 的设置打印如下日志：
+
+```
+global step 100/10008, epoch: 0, batch: 99, rank_id: 0, loss: 2.719922, lr: 0.0000030000, speed: 12.0090 step/s
+eval loss: 2.762143, acc: 0.0532, eval done total : 9.460448503494263 s
+global step 200/10008, epoch: 0, batch: 199, rank_id: 0, loss: 2.536534, lr: 0.0000060000, speed: 5.6834 step/s
+eval loss: 2.326450, acc: 0.26, eval done total : 9.412081480026245 s
+global step 300/10008, epoch: 0, batch: 299, rank_id: 0, loss: 1.847109, lr: 0.0000090000, speed: 5.2913 step/s
+eval loss: 1.447455, acc: 0.471, eval done total : 9.519582033157349 s
+```
```
Comment (tianxin1860) [inline examples/benchmark/clue/README.md:121]: ## 参加 CLUE 竞赛
这一小节教用户怎样做预测，生成符合 CLUE 榜单提交格式的预测结果就 OK

**Review by tianxin1860 — COMMENTED** [review]
Leave some comments

File: examples/benchmark/clue/classification/predict_clue_classifier.py [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+import paddle.nn.functional as F
+from paddle.metric import Metric, Accuracy, Precision, Recall
+
+from paddlenlp.datasets import load_dataset
+from paddlenlp.data import Stack, Tuple, Pad, Dict
+from paddlenlp.transformers import BertForSequenceClassification, BertTokenizer
+from paddlenlp.transformers import ErnieForSequenceClassification, ErnieTokenizer
+from paddlenlp.transformers import RobertaForSequenceClassification, RobertaTokenizer
+
+FORMAT = '%(asctime)s-%(levelname)s: %(message)s'
```
Comment (ZeyuChen) [inline examples/benchmark/clue/classification/predict_clue_classifier.py]: 我理解paddlenlp有统一的log，能去掉这部分代码吗？

File: examples/benchmark/clue/classification/run_clue_classifier.py [hunk 11]
```diff
@@ -0,0 +1,460 @@
...
+from paddle.metric import Accuracy
+
+from paddlenlp.datasets import load_dataset
+from paddlenlp.data import Stack, Tuple, Pad, Dict
+from paddlenlp.transformers import LinearDecayWithWarmup
+from paddlenlp.transformers import BertForSequenceClassification, BertTokenizer
+from paddlenlp.transformers import ErnieForSequenceClassification, ErnieTokenizer
+from paddlenlp.transformers import RobertaForSequenceClassification, RobertaTokenizer
+
+FORMAT = '%(asctime)s-%(levelname)s: %(message)s'
```
Comment (ZeyuChen) [inline examples/benchmark/clue/classification/run_clue_classifier.py]: 这个logger统一成paddlenlp的logger吧

**Review by ZeyuChen — COMMENTED** [review]
去掉那部分logger的代码，改成paddlenlp自己的logger

**Review by tianxin1860 — APPROVED** [review]
LGTM

File: examples/benchmark/clue/classification/predict_clue_classifier.py (hunk 10 above)
Comment (LiuChiachi) [reply examples/benchmark/clue/classification/predict_clue_classifier.py]: 感谢，已经删掉
